bosh-templates/nginx.conf.erb

```diff
@@ -1,7 +1,7 @@
 # deployment cloudcontroller nginx.conf
 user  vcap vcap;
 
-error_log /var/vcap/sys/log/nginx_cc/nginx.error.log;
+error_log <%= p("cc.nginx_error_log_destination") %> <%= p("cc.nginx_error_log_level") %>;
```

Comment:
@sykesm What was the behavior previously with log level if none was specified?  I noticed the default you added on the cf-release PR is `error` https://github.com/cloudfoundry/cf-release/pull/779/files#diff-517e6c86965cf7638732689e362ecb1fR460  so just wanted to know if this is more or less or the same as before.